packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/Chooser.js

```diff
@@ -37,11 +40,13 @@ function Chooser({
         selectedChoiceIds={selectedChoiceIds}
         task={task}
       />
-      <ClearFilters
-        handleFilter={handleFilter}
-        showingChoices={filteredChoiceIds.length}
-        totalChoices={task.choicesOrder?.length}
-      />
+      {showFilters
+        ? (<ClearFilters
+            handleFilter={handleFilter}
+            showingChoices={filteredChoiceIds.length}
+            totalChoices={task.choicesOrder?.length}
+           />)
+        : null}
```

Comment:
Components can wrap other components, so the filters could be wrapped up into something like:
```jsx
<Filters>
  <Choices>
</Filters>
```
where `Filters` could contain the logic for showing the filters.